test/basic.tests.js

```diff
@@ -523,4 +524,172 @@ contract('LiquidityBridgeContract', async accounts => {
         expect(lbcBal).to.be.a.bignumber.eq(initialLPCol);
         expect(web3.utils.toBN(0)).to.be.a.bignumber.eq(finalLPCol);
     });
+
+    it('should register pegout', async () => {
+        const callData = web3.eth.abi.encodeFunctionCall(mock.abi[0], ['99'])
+        const getBalances = () => Promise.all([
+            instance.getBalance("0xf2af87AEB573cdD4B83d543588F85C5280d1cCB5"),
```

Comment:
this address shouldn't be hardcoded. You should get it from the provider.

Reply:
replaced to use one from the accounts array